econml/sklearn_extensions/ensemble.py

```diff
@@ -0,0 +1,722 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Add a corresponding set of unit tests for this file.

Reply:
Done.